swap_meet/vendor.py

```diff
@@ -1,2 +1,90 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        
+        self.inventory = inventory if inventory is not None else []
+
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
+
+
+    def get_by_category(self, category):
+
+        # filter method searches through the inventory and returns a list if the item matches the category
+        category_items = list(filter(lambda x: x.category == category, self.inventory))
```

Comment:
👍 

Love seeing the three approaches. Of the three, the list comprehension would probably be considered the most pythonic.